Src/AwesomeAssertions/Primitives/StringAssertions.cs

```diff
@@ -234,6 +235,126 @@ public AndConstraint<TAssertions> NotBeEquivalentTo(string unexpected,
+        params object[] becauseArgs)
+        where T : IParsable<T>
+    {
+        Guard.ThrowIfArgumentIsNull(formatProvider);
+
+        T parsed = default;
+        var canBeParsed = true;
+        Exception exception = null;
+
+        try
+        {
+            parsed = T.Parse(Subject, formatProvider);
+        }
+        catch (Exception ex) when (ex is OverflowException or FormatException)
+        {
+            canBeParsed = false;
+            exception = ex;
+        }
+
+        CurrentAssertionChain
+            .WithExpectation("Expected {context:the subject} with value {0} to be parseable into {1}{reason}, ",
+                Subject, typeof(T),
+                chain => chain
+                    .BecauseOf(because, becauseArgs)
+                    .ForCondition(canBeParsed)
+                    .FailWith($"but it could not. Exception message: {exception?.Message}"));
+
+        return new AndWhichConstraint<TAssertions, T>((TAssertions)this, parsed);
+    }
+
+    /// <summary>
+    /// Asserts that a string is parseable into something else, which is implementing <see cref="IParsable{TSelf}"/>.
+    /// </summary>
+    /// <param name="because">
+    /// A formatted phrase as is supported by <see cref="string.Format(string,object[])" /> explaining why the assertion
+    /// is needed. If the phrase does not start with the word <i>because</i>, it is prepended automatically.
+    /// </param>
+    /// <param name="becauseArgs">
+    /// Zero or more objects to format using the placeholders in <paramref name="because" />.
+    /// </param>
+    public AndConstraint<TAssertions> NotBeParsableInto<T>(string because = "", params object[] becauseArgs)
+        where T : IParsable<T>
+    {
+        return NotBeParsableInto<T>(CultureInfo.InvariantCulture, because, becauseArgs);
+    }
+
+    /// <summary>
+    /// Asserts that a string isn't parseable into something else, which is implementing <see cref="IParsable{TSelf}"/>,
+    /// with additionally respecting an <see cref="IFormatProvider"/>, e.g. a different <see cref="CultureInfo"/>.
+    /// </summary>
+    /// <param name="because">
+    /// A formatted phrase as is supported by <see cref="string.Format(string,object[])" /> explaining why the assertion
+    /// is needed. If the phrase does not start with the word <i>because</i>, it is prepended automatically.
+    /// </param>
+    /// <param name="becauseArgs">
+    /// Zero or more objects to format using the placeholders in <paramref name="because" />.
+    /// </param>
+    public AndConstraint<TAssertions> NotBeParsableInto<T>(IFormatProvider formatProvider, string because = "",
+        params object[] becauseArgs)
+        where T : IParsable<T>
+    {
+        Guard.ThrowIfArgumentIsNull(formatProvider);
+
+        bool canBeParsed = false;
+
+        try
+        {
+            _ = T.Parse(Subject, formatProvider);
```

Comment:
Are we interested in the exception here? Because in not I will rewrite this to simply `TryParse()`?